### PR TITLE
Updating de Wit+ 2024 abstract w/ html-friendly code and adding Philipp's new paper

### PR DESCRIPTION
**Pull request by sgossage**
Philipp's paper will need to have its NASA ADS link updated when the site properly registers the new paper

### CODE REVIEW

File: papers.html [hunk 1]
```diff
@@ -137,6 +137,26 @@
...
+
+      <p>
+        <button class="btn btn-secondary" type="button" data-toggle="collapse" data-target="#Srivastava20241105" aria-expanded="false" aria-controls="Srivastava20241105">
+          Details
+        </button>
+      </p>
+        <div class="collapse" id="Srivastava20241105">
+          <div class="card card-body">
+            <p><b>Authors: </b> Philipp M. Srivastava, Ugur Demir, Aggelos Katsaggelos, Vicky Kalogera, Elizabeth Teng, Tassos Fragos, Jeff J. Andrews, Simone S. Bavera, Max Briel, Seth Gossage, Konstantinos Kovlakas, Matthias U. Kruckow, Camille Liotine, Kyle A. Rocha, Meng Sun, Zepei Xing, Emmanouil Zapartas</p>
+            <p><b>Access: </b> <a href="https://arxiv.org/abs/2411.02586" target="_blank">arXiv:2411.02586</a> || <a href="https://ui.adsabs.harvard.edu/abs/arXiv:2411.02586" target="_blank">NASA ADS</a></p>
```
Comment (mkruckow) [inline papers.html:154]: The ADS link should become `https://ui.adsabs.harvard.edu/abs/2024arXiv241102586S` according to the ADS naming scheme.

File: papers.html [hunk 2]
```diff
@@ -301,7 +321,7 @@ <h5>de Wit et al. 2024</h5>
         <div class="card card-body">
           <p><b>Authors: </b> de Wit, S.; Bonanos, A. Z.; Antoniadis, K.; Zapartas, E.; Ruiz, A.; Britavskiy, N.; Christodoulou, E.; De, K.; Maravelias, G.; Munoz-Sanchez, G.; Tsopela, A.</p>
           <p><b>Access: </b> <a href="https://arxiv.org/abs/2402.12442" target="_blank">arXiv:2403.2402.12442</a> || <a href="https://ui.adsabs.harvard.edu/abs/2024arXiv240212442D/abstract" target="_blank">NASA ADS</a>
```
Comment (mkruckow) [inline papers.html:323]: `arXiv:2403.2402.12442` should read `arXiv:2402.12442`

File: papers.html (hunk 1 above)
Comment (cplb) [reply papers.html:154]: Neither is in ADS yet

File: papers.html (hunk 2 above)
Comment (cplb) [reply papers.html:323]:
```suggestion
          <p><b>Access: </b> <a href="https://arxiv.org/abs/2402.12442" target="_blank">arXiv:2402.12442</a> || <a href="https://ui.adsabs.harvard.edu/abs/2024arXiv240212442D/abstract" target="_blank">NASA ADS</a> 
```

File: papers.html [hunk 3]
```diff
@@ -301,7 +321,7 @@ <h5>de Wit et al. 2024</h5>
...
...l energy distribution, we obtained the stellar luminosity and radius for 97 RSGs, finding &sim; 50% with log(<i>L</i>/L<sub>&odot;</sub>) &geq; 5.0 and 6 RSGs with <i>R</i> &gtrsim; 1400 R<sub>&odot;</sub>. We also find a correlation between the stellar luminosity and mid-IR excess of 33 dusty, variable sources. Three of these dusty RSGs have luminosities exceeding the revised Humphreys-Davidson limit. We then derive a metallicity-dependent <i>J-K<sub>s</sub></i> color versus temperature relation from synthetic photometry and two new empirical <i>J-K<sub>s</sub></i> color versus temperature relations calibrated on literature TiO and <i>J</i>-band temperatures. To scale our derived, cool TiO temperatures to values in agreement with the evolutionary tracks, we derive two linear scaling relations calibrated on <i>J</i>-band and <i>i</i>-band temperatures. We find that the TiO temperatures are more discrepant as a function of the mass-loss rate and discuss future prospects of the TiO bands as a mass-loss probe. Finally, we speculate that 3 hot, dusty RSGs may have experienced a recent mass ejection (12% of the K-type sample) and indicate them as candidate Levesque-Massey variables.</p>
```
Comment (cplb) [inline papers.html:324]:
```suggestion
          <p><b>Abstract: </b>Mass loss during the red supergiant (RSG) phase plays a crucial role in the evolution of an intermediate massive star, however, the underlying mechanism remains unknown. We aim to increase the sample of well-characterized RSGs at subsolar metallicity, by deriving the physical properties of 127 RSGs in nine nearby, southern galaxies presented by Bonanos et al. For each RSG, we provide spectral types and used MARCS atmospheric models to measure stellar properties from their optical spectra, such as the effective temperature, extinction, and radial velocity. By fitting the spectral energy distribution, we obtained the stellar luminosity and radius for 97 RSGs, finding &sim; 50% with log(<i>L</i>/L<sub>&odot;</sub>) &geq; 5.0 and 6 RSGs with <i>R</i> &gtrsim; 1400 R<sub>&odot;</sub>. We also find a correlation between the stellar luminosity and mid-IR excess of 33 dusty, variable sources. Three of these dusty RSGs have luminosities exceeding the revised Humphreys-Davidson limit. We then derive a metallicity-dependent <i>J-K<sub>s</sub></i> color versus temperature relation from synthetic photometry and two new empirical <i>J-K<sub>s</sub></i> color versus temperature relations calibrated on literature TiO and <i>J</i>-band temperatures. To scale our derived, cool TiO temperatures to values in agreement with the evolutionary tracks, we derive two linear scaling relations calibrated on <i>J</i>-band and <i>i</i>-band temperatures. We find that the TiO temperatures are more discrepant as a function of the mass-loss rate and discuss future prospects of the TiO bands as a mass-loss probe. Finally, we speculate that 3 hot, dusty RSGs may have experienced a recent mass ejection (12% of the K-type sample) and indicate them as candidate Levesque-Massey variables.</p>
```

**Comment by sgossage** [comment]
thank you -- the suggested edits have been incorporated

**Review by tassos25 — APPROVED** [review]

**Review by tassos25 — APPROVED** [review]

**Review by elizabethteng — APPROVED** [review]

**Pull request merged**